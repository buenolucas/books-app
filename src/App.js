import React from 'react';
import styled, {ThemeProvider} from 'styled-components/native';
import Theme from './Theme';

import {RootNavigation} from './Navigation';

const AppContainer = styled.View`
  flex: 1;
  background-color: ${({theme}) => theme.colors.background};
`;

function App() {
  return (
    <ThemeProvider theme={Theme}>
      <AppContainer>
        <RootNavigation />
      </AppContainer>
    </ThemeProvider>
  );
}

export default App;
