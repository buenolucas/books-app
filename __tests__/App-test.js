import 'react-native';
import React from 'react';
import {shallow} from 'enzyme';
import App from '../src/App';

// Note: this is just for use with Jest snapshot testing
// and comes packaged with react-native init project.
// You do not need this if using Enzyme 'toMatchSnapshot' etc.
import renderer from 'react-test-renderer';

// This test just uses Jest snapshot testing
it('renders correctly, test using Jest', () => {
  renderer.create(<App />);
});

// Using Jest + Enzyme
describe('<App />', () => {
  it('renders correctly, test using Jest + Enzyme', () => {
    expect(shallow(<App />)).toMatchSnapshot();
  });
});
