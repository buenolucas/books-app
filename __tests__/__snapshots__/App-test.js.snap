// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly, test using Jest + Enzyme 1`] = `ShallowWrapper {}`;
